packages/office-addin-sso/src/middle-tier/app.ts

```diff
@@ -76,19 +74,32 @@ export class App {
     });
 
     this.appInstance.get("/getuserdata", async function (req: any, res: any, next: any) {
```

Comment:
The access token passed in from the client must be validated here.

Reply:
I believe I've added the validation correctly.

Reply:
My bad, I see it now. thanks!